source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,138 +1,52 @@
-
-If Fastly goes down, we would manually [switch to AWS CloudFront](/manual/fall-back-to-aws-cloudfront.html) instead of Fastly. Where Fastly makes requests to GOV.UK Origin, AWS CloudFront instead makes all its requests to the GOV.UK mirror.
+- HTML pages
+- related assets for those pages (e.g. JavaScript, CSS, images, fonts) HTML page and asset files.
+- other linked assets (or "attachments") such as CSVs, PDFs etc.
 
-## GOV.UK mirror locations and access
+## Available mirrors
 
-The GOV.UK mirror is hosted in an [Amazon Web Services (AWS) S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). The bucket contains copies of GOV.UK HTML files. The mirror is static, meaning dynamic pages such as search pages will not work. S3 will return 403 Forbidden response for non-existent pages instead of 404, because we don't allow the ListBucket permission. This also affects requests for features such as Search, which are not covered by the mirrors.
+We maintain three mirrors, ranked by priority:
 
-The term "GOV.UK mirror" actually refers to 3 separate mirrors:
+1. Primary: AWS S3 bucket named `govuk-<environment>-mirror` in eu-west-2
+1. Secondary: AWS S3 bucket named `govuk-<environment>-mirror-replica` in eu-west-1
+1. Tertiary: Google Cloud Storage (GCP) bucket named `govuk-<environment>-mirror`
 
-- the main `govuk-<environment>-mirror` S3 bucket in one AWS region
-- the `govuk-<environment>-mirror-replica` S3 bucket in another region, so if the first AWS region is down, we can fall back to this other region
-- the `govuk-<environment>-mirror` bucket in Google Cloud Storage (GCS), so if AWS overall is down, we can fall back to this mirror
+We use multiple mirrors across various AWS regions and GCP to ensure redundancy and increase availability.
 
-Access to the Amazon S3 buckets is restricted. If you have a Fastly, Office or Pingdom IP address, you have read-only access. If you’re an authenticated AWS web console user, you have read-write access.
-
-Access to the GCS bucket is also restricted. You can access the GCS bucket if you can access the secret keys in `govuk-secrets`, or if you’re an authenticated Google Cloud Platform (GCP) web console user.
-
-## Updates to the GOV.UK mirror
-
-Automatic scripts update the GOV.UK mirror every day.
+## When is the GOV.UK mirror used?
 
-The [`govuk_seed_crawler`](https://github.com/alphagov/govuk_seed_crawler) is a script on the GOV.UK “mirrorer” machine. Every day at 8:00pm, this script fetches all of the URLs in the [GOV.UK sitemap](https://www.gov.uk/sitemap.xml) and adds them to a [RabbitMQ message queue](/manual/rabbitmq.html).
+If [Fastly, our primary CDN](/manual/cdn.html), cannot fetch a page from our backend servers (due to timeouts or 5xx errors), then Fastly will attempt to serve a page from a mirror in order of priority.
```

Comment:
```suggestion
If [Fastly, our primary CDN](/manual/cdn.html), cannot fetch a page from our backend servers (because of a timeout or 5xx error), then Fastly will attempt to serve a page from a mirror in order of priority.
```